src/main/java/com/milamber_brass/brass_armory/item/ICustomReachItem.java

```diff
@@ -0,0 +1,54 @@
+package com.milamber_brass.brass_armory.item;
+
+import com.google.common.collect.ImmutableMultimap;
+import com.google.common.collect.ImmutableMultimap.Builder;
+import com.google.common.collect.Multimap;
+import net.minecraft.entity.ai.attributes.Attribute;
+import net.minecraft.entity.ai.attributes.AttributeModifier;
+import net.minecraft.entity.ai.attributes.AttributeModifier.Operation;
+import net.minecraft.inventory.EquipmentSlotType;
+import net.minecraft.item.ItemStack;
+import net.minecraftforge.common.ForgeMod;
+import net.minecraftforge.common.extensions.IForgeItem;
+
+import javax.annotation.Nonnull;
+import java.util.UUID;
+
+public interface ICustomReachItem extends IForgeItem {
+
+    //Needs to be unique
+    public static final UUID REACH_DISTANCE_MODIFIER = UUID.fromString("b7d3df52-d360-491b-9bb5-2e8e3b5b279a");
```

Comment:
These were changed by the formatter, apologies for removing them. No need to be passive aggressive with commit messages.